src/test/java/com/fasterxml/jackson/databind/objectid/TestObjectIdWithEquals.java

```diff
@@ -101,8 +101,8 @@ public void testSimpleEquals() throws Exception
         foo.otherBars.add(bar2);
 
         String json = mapper.writeValueAsString(foo);
-        assertEquals("{\"id\":1,\"bars\":[{\"id\":1},{\"id\":2}],\"otherBars\":[1,2]}", json);
-
+        assertTrue(json.equals("{\"id\":1,\"bars\":[{\"id\":1},{\"id\":2}],\"otherBars\":[1,2]}")
```

Comment:
Given number of fields this is not sufficient. But also not needed: please add `@JsonPropertyOrder` declaration for `Foo` class instead to force serialization ordering

Reply:
> Given number of fields this is not sufficient. But also not needed: please add `@JsonPropertyOrder` declaration for `Foo` class instead to force serialization ordering

Thank you, I have updated the PR